src/Skills.jsx

```diff
@@ -78,7 +79,7 @@ class Skills extends Component {
       })
       .catch(err => {
         console.log(err);
-      });
+      }).then(this.loadReviews);
```

Comment:
Should `loadReviews` be placed in the above `then` statement? It also seems like to run the second `then` function after the `catch`, too.